extensions/exquery/modules/pom.xml

```diff
@@ -15,15 +15,14 @@
     <name>eXist-db EXQuery Extension Modules</name>
     <description>eXist-db NoSQL Database EXQuery Extension Modules</description>
 
-    <scm>
-        <connection>scm:git:https://github.com/exist-db/exist.git</connection>
-        <developerConnection>scm:git:https://github.com/exist-db/exist.git</developerConnection>
-        <url>scm:git:https://github.com/exist-db/exist.git</url>
-      <tag>HEAD</tag>
-  </scm>
-
     <modules>
         <module>request</module>
     </modules>
 
+    <scm>
```

Comment:
Why does this come after the modules? surely one of the most important things is knowing where the source code is?